xbmc/filesystem/CurlFile.cpp

```diff
@@ -44,16 +44,20 @@ using namespace std::chrono_literals;
 
 #define FITS_INT(a) (((a) <= INT_MAX) && ((a) >= INT_MIN))
 
-static const auto proxyType2CUrlProxyType = std::unordered_map<XFILE::CCurlFile::ProxyType, int>{
+namespace
+{
+const auto proxyType2CUrlProxyType{std::unordered_map<XFILE::CCurlFile::ProxyType, int>{
```

Comment:
If you feel like it you can also switch this to our own `CMap` and make it `constexpr`:
```cpp
constexpr auto proxyType2CUrlProxyType{make_map<XFILE::CCurlFile::ProxyType, int>({
    {CCurlFile::ProxyType::HTTP, CURLPROXY_HTTP},
    {CCurlFile::ProxyType::SOCKS4, CURLPROXY_SOCKS4},
    {CCurlFile::ProxyType::SOCKS4A, CURLPROXY_SOCKS4A},
    {CCurlFile::ProxyType::SOCKS5, CURLPROXY_SOCKS5},
    {CCurlFile::ProxyType::SOCKS5_REMOTE, CURLPROXY_SOCKS5_HOSTNAME},
    {CCurlFile::ProxyType::HTTPS, CURLPROXY_HTTPS},
})};
```

Reply:
Will do so, thanks.